starter-code/routes/passportRouter.js

```diff
@@ -0,0 +1,170 @@
+const express = require("express");
+const passportRouter = express.Router();
+
+// Require user model
+const User = require("../models/user");
+
+// Add bcrypt to encrypt passwords
+const bcrypt = require("bcrypt");
+
+// Add ensureLogin
+const ensureLogin = require("connect-ensure-login").ensureLoggedIn;
+
+// Add passport
+const passport = require("passport");
+
+// Add our checkRoles middleware
+const checkRoles = require("../auth/checkRoles");
+
+//// Routes
+
+// Get overview of all the users
+passportRouter.get("/employees", ensureLogin("/login"), (req, res, next) => {
+  let allowedEdits = false;
+  User.find({})
+    .then(user => {
+      if (req.user.role === "BOSS") {
+        allowedEdits = true;
+      }
+      res.render("passport/employees", { user, currentUser: req.user.username, allowed: allowedEdits });
+    })
+    .catch(e => next(e));
+});
+
+// Edit user - GET
+passportRouter.get("/editemployee/:id", checkRoles("BOSS"), (req, res, next) => {
+  User.findById(req.params.id)
+    .then(user => {
+      res.render("passport/editEmployee", { user, role: user.role });
```

Comment:
Since you're already passing the user role via "user", "role" isn't needed here.